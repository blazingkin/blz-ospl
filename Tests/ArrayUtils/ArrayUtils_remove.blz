:main
    #Test 00 :remove
	print(">>> Test 00 :remove <<<")
	arr = [0,1,2,3,4,5,6]
	print("before remove: " + arr)
	arr = arr.remove(3)
	print("after removing index 3: " + arr)
	arr = arr.remove(5)
	print("after removing index 5: " + arr)
	arr = arr.remove(0)
	print("after removing index 0: " + arr)
	arr2 = arr.remove(1)
	print("arr2 makes a copy of arr and removes index 1: " + arr2)
	print("--------------------------------")

	#Test 01 :remove!
	#print(">>> Test 01 :remove! <<<")
	#arr = [0,1,2,3,4,5,6]
	#print("before remove: " + arr)
	#arr.remove!(3)
	#print("after removing index 3: " + arr)
	#arr.remove!(5)
	#print("after removing index 5: " + arr)
	#arr.remove!(0)
	#print("after removing index 0: " + arr)
	#arr2 = arr.remove!(1)
	#print("arr2 makes a copy of arr and removes index 1: " + arr2)
	#print("--------------------------------")

	# >>> POSSIBLE CALLS <<<
	#	arr = arr.remove(index)
	#	arr.remove(index)
	#	arr = arr.remove!(index)
	#	arr.remove!(index)
end