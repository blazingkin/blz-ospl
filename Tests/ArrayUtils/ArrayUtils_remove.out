>>> Test 00 :remove <<<
before remove: [0, 1, 2, 3, 4, 5, 6]
after removing index 3: [0, 1, 2, 4, 5, 6]
after removing index 5: [0, 1, 2, 4, 5]
after removing index 0: [1, 2, 4, 5]
arr2 makes a copy of arr and removes index 1: [1, 4, 5]
--------------------------------
